package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "ember-snippets",
+  "publisher": "ember-tooling",
+  "displayName": "Ember Polaris Snippets - gjs, gts, tests, services, etc",
+  "description": "Snippets for Ember projects for quickly generating boilerplate",
+  "icon": "images/ember-e-circle-icon-4c.png",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ember-tooling/snippets.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ember-tooling/snippets/issues"
+  },
+  "homepage": "https://github.com/ember-tooling/snippets#readme",
+  "version": "0.0.0",
+  "categories": [
+    "Snippets"
+  ],
+  "engines": {
+    "vscode": ">=1.0.0"
+  },
+  "keywords": [
+    "Ember",
+    "TypeScript",
+    "JavaScript",
+    "Snippets",
+    "Polaris",
+    "Tests",
+    "Testing",
+    "Services",
+    "Boilerplate",
+    "HTML"
+  ],
+  "contributes": {
+    "snippets": [
+      {
+        "language": [
+          "Glimmer JS",
+          "javascript.glimmer",
+          "Glimmer TS",
+          "typescript.glimmer",
+          "javascript",
+          "typescript"
+        ],
+        "path": "./snippets/all.json"
+      },
+      {
+        "language": [
+          "Glimmer JS",
+          "javascript.glimmer",
+          "Glimmer TS",
+          "typescript.glimmer"
+        ],
+        "path": "./snippets/all-template-tag.json"
+      },
+      {
+        "language": [
+          "Glimmer JS",
+          "javascript.glimmer"
+        ],
+        "path": "./snippets/javascript.glimmer.json"
+      },
+      {
+        "language": [
+          "Glimmer TS",
+          "typescript.glimmer"
+        ],
+        "path": "./snippets/typescript.glimmer.json"
+      },
+      {
+        "language": "javascript",
+        "path": "./snippets/javascript.json"
+      },
+      {
+        "language": "typescript",
+        "path": "./snippets/typescript.json"
+      }
+    ]
+  },
+  "scripts": {
+    "lint:fix": "prettier . --write",
+    "build": "vsce pack",
+    "publish": "npm version patch; vsce publish"
```

Comment:
other addons use these scripts

```json
  "vs:package": "vsce package --yarn",
  "vs:publish": "vsce publish --yarn",
  "vs:publish:ci": "vsce publish --yarn --pat $VSCODE_TOKEN",
  "ov:publish:ci": "ovsx publish --yarn --pat $OPENVSX_TOKEN",
```
Will need to add the ovsx package.
I'd assume we can just drop the --yarn flag since this uses pnpm.

Reply:
done